Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -169,6 +169,15 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging {
                         to parentViewController: UIViewController,
                         pointAnnotationManager: PointAnnotationManager)
     
+    /**
+     Offers the delegate an opportunity to use a customized rounding mechanism for the remaining distance.
+     
+     - parameter carPlayManager: The CarPlay manager instance.
+     - parameter remainingDistance: The distance remaining to be rounded.
+     - returns: An optional value representing the distance remaining.
+     */
```

Comment:
Probably it should be:
> The remaining distance to be rounded
> An optional value representing the remaining distance

Reply:
ah thank you. i will update this comment.